app/lib/reference_library_document.rb

```diff
@@ -32,32 +34,33 @@ def self.new_from_csv(title, description, author, date, relevant_pages, download
       extract_download_url(download_url),
       extract_download_url(thumbnail_url),
       technical_area,
-      resource_type
+      reference_type
     )
   end
 
   ##
   # if there are multiple then it returns only the last which would have been the first-added attachment.
   # example arg: "One health zoonotic disease prioroitization for Multisectoral Engagement in Burkina Faso (FRENCH).pdf (https://dl.airtable.com/.attachments/d273fbb3427d94635ddb734a14f95e26/98c1a677/OnehealthzoonoticdiseaseprioroitizationforMultisectoralEngagementinBurkinaFasoFRENCH.pdf),One health zoonotic disease prioroitization for Multisectoral Engagement in Burkina Faso.pdf (https://dl.airtable.com/.attachments/a78a3c2b524ddac30133b6aa882817aa/15400a04/OnehealthzoonoticdiseaseprioroitizationforMultisectoralEngagementinBurkinaFaso.pdf)"
   def self.extract_download_url(attachments_field)
-    return "" if attachments_field.blank?
+    return '' if attachments_field.blank?
+
     # tokenize into an array based on comma. most have one but some have two.
-    attachments = attachments_field.split(",")
+    attachments = attachments_field.split(',')
```

Comment:
NIT: I switched from single quotes to double quotes because, as I recall, double quotes in Ruby code is the rubocop/community best practice. I could go either way on which we use, but I don't think we should be going back and forth.

Reply:
I didn't change this manually, our current rubocop config has single quotes set from what I can tell (we don't have a rubocop.yml file, so it defaults to this: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/StringLiterals)

Reply:
no preference either way though, happy to do what rubocop tells me to :)